design-documents/graph-ql/coverage/catalog/compare-list.graphqls

```diff
@@ -0,0 +1,38 @@
+type ComparableItem {
+    productId: ID!
+    name: String!
+    priceRange: ProductPriceRange!
+    url: ProductUrl
+    images: [ProductImage]
+    values: [ProductAttribute]
+}
+
+type ComparableIndicator {
+    code: String!
+    title: String!
+}
+
+type CompareList {
+    items: [ComparableItem]
+    indicators: [ComparableIndicator]
+}
+
+type Customer{
+    compareList: CompareList
+}
+
+input CompareListInput {
+    id: ID!
+    items: [ID!]
+}
+
+type Mutation {
+    addItemsToCompareList(list: CompareListInput): CompareList
```

Comment:
It is better to explicitly list all arguments without wrapper.

Reply:
Fixed